packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomFooter.Example.tsx

```diff
@@ -71,14 +71,17 @@ export class DetailsListCustomFooterExample extends React.Component<{}, {}> {
   }
 }
 
-function _renderDetailsFooterItemColumn(item: IDetailsListCustomFooterExampleItem, index: number, column: IColumn) {
-  return (
-    <div>
-      <b>{column.name}</b>
-    </div>
-  );
-}
+const _renderDetailsFooterItemColumn: IDetailsRowBaseProps['onRenderItemColumn'] = (item, index, column) => {
```

Comment:
Just to clarify, changing this to const is making this usage in the examples such that the props are required and not optional like before?

Reply:
the const is just to be consistent in our examples of using const + lambdas for internal functions. Also makes it easier to show the typing.

But yes, the big change is that I refer to the exact signature for onRenderItemColumn vs simply trying to recreate it